packages/components/molecules/f-tabs/src/components/Tabs.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div
+        :class="$style['c-tabs']"
+        data-test-id='tabs-component'>
```

Comment:
Minor, should be double quotes here.

Reply:
generated this way, I will make the change

Reply:
I've updated this in the generator now too 👍